File: servers/scalatra/src/main/scala/com/wordnik/client/model/quote.scala
package com.wordnik.client.model

import java.util.Date
import java.math.BigDecimal


case class Quote (
  timestamp: Date,
  symbol: String,
  bidSize: BigDecimal,
  bidPrice: Double,
  askPrice: Double,
  askSize: BigDecimal,
  id: Double
)

File: clients/typescript-angular/API/Client/Quote.ts
/// <reference path="api.d.ts" />

namespace API.Client {
    'use strict';

    export interface Quote {

        timestamp?: date;

        symbol?: string;

        bidSize?: number;

        bidPrice?: number;

        askPrice?: number;

        askSize?: number;

        id?: number;
    }

}
